client/state/sites/reducer.js

```diff
@@ -23,19 +24,20 @@ import { SITE_RECEIVE, SERIALIZE, DESERIALIZE } from 'state/action-types';
 export function items( state = {}, action ) {
 	switch ( action.type ) {
 		case SITE_RECEIVE:
+			//TODO: do not pass a decorated site object to SITE_RECEIVE
+			//site objects are being decorated in SitesList in lib/sites/sites-list/index.js
+			//with either lib/site/index.js or lib/site/jetpack.js
+			const blackList = [ '_headers', 'fetchingSettings', 'fetchingUsers',
```

Comment:
I was hoping to fix this properly soon by passing the resource before it's decorated to fix #2757